ipld/hamt/Cargo.toml

```diff
@@ -8,16 +8,16 @@ edition = "2018"
 repository = "https://github.com/ChainSafe/forest"
 
 [dependencies]
-forest_encoding = "0.2.1"
+forest_encoding = "0.2"
 serde = { version = "1.0", features = ["derive"] }
-byteorder = "1.3.2"
+byteorder = "1.3"
 cid = { package = "forest_cid", version = "0.3" }
 db = { package = "forest_db", version = "0.1" }
 ipld_blockstore = "0.1"
-forest_ipld = "0.1.1"
-serde_bytes = { package = "cs_serde_bytes", version = "0.12" }
+forest_ipld = "0.1"
+serde_bytes = { version = "0.11" }
```

Comment:
Should be fine with just `serde_bytes = "0.11"`, right? Same in the other Cargo file. Unless you feel it's valuable to leave it explicit here, then it's fine for me. :)